CMakeLists.txt

```diff
@@ -2,7 +2,13 @@ cmake_minimum_required (VERSION 2.8)
 
 project(OpenNI2)
 
-add_subdirectory(libusbx)
+add_definitions(-UUNICODE)
+add_definitions(-U_UNICODE)
+
+if(APPLE)
```

Comment:
might be more accurate to do if(NOT WIN32), because I think Linux will use libusbx, too.